src/util/utils.js

```diff
@@ -58,4 +58,14 @@ function isArray(thing) {
     return (Object.prototype.toString.call(thing) === '[object Array]');
 }
 
-export {isFunction, isUndefined, isDefined, isObject, isArray, isString};
+/**
+ * Remove decimal place if the decimal value is empty.
+ *
+ * @param value
+ * @returns {*}
+ */
+function precision(value) {
+    return value % 1 === 0 ? 0 : null;
```

Comment:
The `precision` is for removing the decimal points only if it's 0. Otherwise, it always returns a value with two decimal points.